qemu/tests/watchdog.py

```diff
@@ -282,6 +282,30 @@ def hotplug_unplug_watchdog_device():
                 test.fail("Oops, find watchdog pci, unplug failed")
             logging.info("The WDT remove successfully")
 
+    def stop_cont_test():
+        """
+        Check if the emulated watchdog devices work properly with the stop/
+        continue operation
+        """
+
+        response_timeout = int(params.get("response_timeout", '240'))
+        _watchdog_device_check(test, session, watchdog_device_type)
+        _trigger_watchdog(session, trigger_cmd)
+        vm.pause()
+        vm.monitor.clear_event("WATCHDOG")
```

Comment:
Better clean it before trigger watch dog.

Reply:
Agree with above.